xml/storage_multipath.xml

```diff
@@ -2489,723 +2406,682 @@ blacklist_exceptions {
      }
 }</screen>
-     <para>
-       The <literal>blacklist_exceptions</literal> section supports all
-       methods described for the <literal>blacklist</literal> section above.
-     </para>
-     <para>
-       The <literal>property</literal> directive in
-       <literal>blacklist_exceptions</literal> is special, because every
-       device <emphasis>must</emphasis> have at least one of the
-       &ldquo;whitelisted&rdquo; udev properties in order to be condsidered as
-       path device for multipath (the value of the property does not matter).
-       The built-in default for <literal>property</literal> is
-     </para>
-     <screen>    property (SCSI_IDENT_|ID_WWN)</screen>
-     <para>
-       Only devices that have at least one udev property matching this
-       regular expression will be included.
-     </para>
-   </sect2>
+   <para>
+    The <literal>blacklist_exceptions</literal> section supports all methods
+    described for the <literal>blacklist</literal> section above.
+   </para>
+   <para>
+    The <literal>property</literal> directive in
+    <literal>blacklist_exceptions</literal> is mandatory because every device
+    <emphasis>must</emphasis> have at least one of the &ldquo;allowed&rdquo;
+    udev properties to be considered a path device for multipath
+    (the value of the property does not matter). The built-in default for
+    <literal>property</literal> is
+   </para>
+<screen>    property (SCSI_IDENT_|ID_WWN)</screen>
+   <para>
+    Only devices that have at least one udev property matching this regular
+    expression will be included.
+   </para>
+  </sect2>
 
-   <sect2 xml:id="sec-multipath-device-select-other">
-     <title>Other options affecting device selection</title>
-     <para>
-       Besides the blacklisting options, a few other settings in
-       <filename>/etc/multipath.conf</filename> affect which devices are
-       considered multipath path devices.
-     </para>
-     <variablelist>
+  <sect2 xml:id="sec-multipath-device-select-other">
+   <title>Other options affecting device selection</title>
+   <para>
+    Besides the <literal>blacklist</literal> options, there are several other
+    settings in <filename>/etc/multipath.conf</filename> that affect which
+    devices are considered multipath path devices.
+   </para>
+   <variablelist>
+    <varlistentry>
+     <term>find_multipaths</term>
+     <listitem>
+      <para>
+       This option controls the behavior of <command>multipath</command> and
+       <command>multipathd</command> when a device that is not excluded is
+       first encountered. The possible values are:
+      </para>
+      <variablelist>
+       <varlistentry>
+        <term>greedy</term>
+        <listitem>
+         <para>
+          Every non-excluded device is included. This is the default on &sle;.
+          If this setting is active, the only way to prevent devices from being
+          added to multipath maps is setting them as excluded.
```

Comment:
We should keep the term "blacklisting" here. I know it's considered politically incorrect these days, but it's the technical term used in the the config file, and we should be precise. "excluded" isn't 100% unambiguous. If you must keep "excluded", please add an xref to make sure readers understand what we mean.